CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ SET(PROJECT_DESCRIPTION
   )
 SET(PROJECT_URL "https://github.com/humanoid-path-planner/hpp-core")
 
+OPTION(INSTALL_DOCUMENTATION "Generate and install the documentation" ON)
+
```

Comment:
For info:
- this must be called before SETUP_[HPP_]PROJECT.
- shouldn't this be done either:
  - here https://github.com/jrl-umi3218/jrl-cmakemodules/blob/320c636960b03b3bad7c7a08bd2e104951f42bc3/base.cmake#L216
  - here https://github.com/jrl-umi3218/jrl-cmakemodules/blob/320c636960b03b3bad7c7a08bd2e104951f42bc3/doxygen.cmake#L15
  - or in SETUP_HPP_PROJECT
?